src/lmic/lmic.c

```diff
@@ -295,7 +295,7 @@ ostime_t calcAirTime (rps_t rps, u1_t plen) {
 //
 
 static void setRxsyms (ostime_t rxsyms) {
-    if (rxsyms >= (1u << 10u)) {
+    if (rxsyms >= (ostime_t)(1 << 10u)) {
```

Comment:
I disagree with this change. The (ostime-t) does nothing , because the shift is performed usign the type of hte left operand. If any chage is made, this must be: `(((ostime_t) 1) << 10u)` (i.e., use ostime_t as the left operand). 

Since this is a constant calculation, it doesn't matter, but if someone changes 1 to a variable, code will still be correct. If someone cuts and pastes and changes, etc.)

Reply:
Thanks for your review !
I changed it to `if (rxsyms >= (((ostime_t)1) << 10u))`